dist/react-datepicker.js

```diff
@@ -440,6 +440,15 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	    var timeFormats = ["HH:mm:ss a"];
 
+	    var manualDateString = this.state.manualDate;
+
+	    // Add a space between a number and pm if needed
+	    if (manualDateString) {
+	      for (var i = 0; i <= 9; i++) {
+	        manualDateString = manualDateString.replace(new RegExp(i.toString() + "pm", "g"), i.toString() + " pm");
+	      }
```

Comment:
How do you do a regex replace such that it doesn't replace the number as well? I was hoping I could replace it with "[0-9] pm" but that doesn't work.